src/con_duct/__main__.py

```diff
@@ -345,24 +345,24 @@ def execution_summary(self) -> dict[str, Any]:
             "exit_code": self.process.returncode,
             "command": self.command,
             "logs_prefix": self.log_paths.prefix,
-            "wall_clock_time": f"{self.elapsed_time:.3f} sec",
-            "peak_rss": (
-                f"{self.max_values.total_rss} KiB"
+            "wall_clock_time": f"{self.elapsed_time:.3f}",
+            "peak_rss_kib": (
```

Comment:
I think we better just report in bytes. Check ps options or convert if needed from kbs